### PR TITLE
Renamed sound enums for accuracy and organization

### PR DESCRIPTION
**Pull request by Generalcamo**
Per the request of @Gymnasiast, renamed the sound enums for better accuracy.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
We could look into switching this to an enum class as well whilst your at it.

File: src/openrct2/audio/Audio.cpp [hunk 1]
```diff
@@ -59,9 +59,9 @@ rct_vehicle_sound_params* gVehicleSoundParamsListEnd;
 // clang-format off
 static int32_t SoundVolumeAdjust[SOUND_MAXID] =
 {
-    0,      // SOUND_LIFT_1
-    0,      // SOUND_TRACK_FRICTION_1
-    0,      // SOUND_LIFT_2
+    0,      // SOUND_LIFT_CLASSIC
+    0,      // SOUND_TRACK_FRICTION_CLASSIC_WOOD
+    0,      // SOUND_FRICTION_CLASSIC
```
Comment (Gymnasiast) [inline src/openrct2/audio/Audio.cpp]: I suppose you mean friction wheels? That's indeed one usage (the Junior RC, for example), but it's also used by rides like the dinghy slide, which uses a conveyor belt.

File: src/openrct2/audio/Audio.cpp [hunk 2]
```diff
@@ -71,27 +71,27 @@ static int32_t SoundVolumeAdjust[SOUND_MAXID] =
     0,      // SOUND_SCREAM_4
     0,      // SOUND_SCREAM_5
     0,      // SOUND_SCREAM_6
-    0,      // SOUND_LIFT_3
+    0,      // SOUND_LIFT_VEKOMA
```
Comment (Gymnasiast) [inline src/openrct2/audio/Audio.cpp]: Hm, why Vekoma? Isn't this sound also used for the Junior RC and dinghy slide?

File: src/openrct2/audio/audio.h [hunk 3]
```diff
@@ -92,27 +92,27 @@ enum RCT2_SOUND
...
     SOUND_COUGH_3,
     SOUND_COUGH_4,
-    SOUND_RAIN_1,
+    SOUND_RAIN,
     SOUND_THUNDER_1,
     SOUND_THUNDER_2,
-    SOUND_RAIN_2,
-    SOUND_RAIN_3,
+    SOUND_TRACK_FRICTION_TRAIN,
+    SOUND_TRACK_FRICTION_WATER,
```
Comment (Gymnasiast) [inline src/openrct2/audio/audio.h]: Could you clarify this one? You might be right, but I don't immediately recognise the sound.

**Comment by Gymnasiast** [comment]
Most of these look all right. I have put line notes on two of them, though.

File: src/openrct2/audio/Audio.cpp (hunk 2 above)
Comment (Generalcamo) [reply src/openrct2/audio/Audio.cpp]: Yes it is, though the sound is used in odd places anyway. Per discussion, I'll rename it FRICTION_WHEEL for now.

File: src/openrct2/audio/audio.h (hunk 3 above)
Comment (Generalcamo) [reply src/openrct2/audio/audio.h]: This is used for most of the water rides. Examples include the River Rapids and the Dinghy Slide:
https://github.com/OpenRCT2/objects/blob/72dfb9adc551a0b5a2e173dcf6536402e9a5acb0/objects/rct2/ride/rct2.ding1.json#L32

Exceptions include the river rafts.

File: src/openrct2/audio/Audio.cpp (hunk 2 above)
Comment (Gymnasiast) [reply src/openrct2/audio/Audio.cpp]: `FRICTION_WHEELS`, they always come in pairs ;)

**Pull request merged**